packages/suite/src/components/wallet/AccountsMenu/index.tsx

```diff
@@ -189,6 +189,14 @@ export const AccountsMenu = ({ isMenuInline }: AccountsMenuProps) => {
         a => a.accountType === 'ledger' && (!a.empty || a.visible),
     );
 
+    const bip44ChangeAccounts = filteredAccounts.filter(
+        a => a.accountType === 'bip44Change' && (!a.empty || a.visible),
+    );
```

Comment:
I am very unfamiliar with how solana works - but is this change similar to change chain in bitcoin-like coins? if yes, does it make sense to have this as a "top level" account?

Reply:
Yeah, please let us know. 🙏🏻
The change and root derivation path is weird. I believe that the path m/44’/501’/i’ is the one that is used the most, right?

Reply:
We have this info from discussions with the Solana guys
```
So m/44'/501', m/44'/501'/i' are both legacy, solely used by solflare and the solana cli. m/44'/501'/i'/0' technically is the standard that most wallets are using
```

We started a discussion with solana guys again, asking would would make sense to them to be supported in the trezor-suite, again I will let you know.

Reply:
If you guys decide to use just a single path, solana guys recommend using `m/44'/501'/i'/0' `